include/shell.h

```diff
@@ -180,10 +199,7 @@ class DOS_Shell : public Program, public HostShell {
 	void CMD_VOL(char* args);
 	void CMD_MOVE(char* args);
 
-	/* The shell's variables */
-	uint16_t input_handle         = 0;
-	bool echo                     = false;
-	bool call                     = false;
+	bool echo = true;
```

Comment:
Does this mean we're changing the default behaviour? If so, why? Was our previous behaviour incorrect?